nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -18,6 +18,15 @@
 
 - [sitespeed-io](https://sitespeed.io), a tool that can generate metrics (timings, diagnostics) for websites. Available as [services.sitespeed-io](#opt-services.sitespeed-io.enable).
 
+- Support for WiFi6 (IEEE 802.11ax) and WPA3-SAE-PK was enabled in the `hostapd` package.
+
+- The `services.hostapd` module was rewritten to support `passwordFile` style options, configuration of WPA3-SAE, and management of multiple interfaces.
+
+  - `hostapd` is now started with additional systemd sandbox/hardening options for better security.
+  - `services.hostapd.interface` was replaced with a per-radio and per-bss configuration scheme using [services.hostapd.radios](#opt-services.hostapd.radios).
+  - `services.hostapd.wpa` has been replaced by [services.hostapd.radios.&lt;name&gt;.networks.&lt;name&gt;.authentication.wpaPassword](#opt-services.hostapd.radios._name_.networks._name_.authentication.wpaPassword) and [services.hostapd.radios.&lt;name&gt;.networks.&lt;name&gt;.authentication.saePasswords](#opt-services.hostapd.radios._name_.networks._name_.authentication.saePasswords) which configure passwords WPA2-PSK and WP3-SAE respectively.
+  - The default authentication has been changed to WPA3-SAE. Options for other (legacy) schemes are stil availabe.
```

Comment:
We can't just tell it to use the best supported scheme?

Reply:
No. You can of course choose wpa3 transition if you want (allowing both sae and wpa2 if the client doesn't support sae3) but this of course allows for downgrade attacks. Since almost all devices can do sae today just fine, it should be the default.